modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/BaseWorkflowDefinitionLinkResourceTestCase.java

```diff
@@ -94,6 +96,8 @@ public void setUp() throws Exception {
 		testCompany = CompanyLocalServiceUtil.getCompany(
 			testGroup.getCompanyId());
 
+		_workflowDefinition = WorkflowDefinitionTestUtil.addWorkflowDefinition();
```

Comment:
Hi @victorKammerer, can you make those changes to the file `WorkflowDefinitionLinkResourceTest.java`? and leave this file as it was generated.

Reply:
Done, @linolaoj!